docs/src/content/docs/ops/guides/starting-with-gcp.adoc

```diff
@@ -0,0 +1,258 @@
+
+== Deploying the Google Cloud Infrastructure
+
+Before creating the Terraform resources, review and update the `terraform.tfvars` file to ensure the parameters are correctly set for your environment:
+
+* You are **required** to set the `project_id` parameter to the Google Cloud project ID you wish to deploy into. 
+* You may also wish to change resource tiers, the location of the resource group, or the VM sizes used by the Google Cloud cluster.
+
+To get a full list of the resources that will be deployed by the templates, run:
+```bash
+terraform plan
+```
+
+Finally, to create the resources, run:
+```bash
+terraform apply
+```
+
+This will create the necessary infrastructure on the Google Cloud Project.
+
+[#terraform-outputs]
+=== Fetching the Terraform Outputs
+
+The Terraform templates will generate several outputs required for setting up the XTDB nodes on the GKE cluster.
+
+To retrieve these outputs, execute the following command:
+```bash
+terraform output
+```
+
+This will return the following outputs:
+
+* `project_id` - The Google Cloud project ID.
+* `bucket_name` - The name of the Google Cloud Storage bucket.
+* `service_account_email` - The email address of the service account.
+* `kubernetes_namespace` - The (expected) Kubernetes namespace in which the XTDB resources will be deployed.
+* `kubernetes_service_account_name` - The (expected) name of the Kubernetes service account used by the XTDB nodes.
+
+NOTE: Within the deployed terraform infrastructure, we setup link:https://cloud.google.com/kubernetes-engine/docs/concepts/workload-identity[**Workload Identity Federation**^] on the Google Cloud IAM service account. 
+This will allow the service account to be used within the GKE cluster, specifically by the `kubernetes_service_account_name` service account under the `kubernetes_namespace` namespace (`xtdb-service-account`, under `xtdb-deployment`, by default).
+
+== Deploying on Kubernetes
+
+With the infrastructure created on Google Cloud, we can now deploy the XTDB nodes and a simple Kafka instance on the Google Kubernetes Engine cluster.
+
+Prior to deploying the Kubernetes resources, ensure that the `kubectl` CLI is installed and configured to interact with the GKE cluster. Run the following command:
+
+```bash
+gcloud container clusters get-credentials xtdb-cluster --region us-central1
+```
+
+Now that `kubectl` is authenticated with the GKE cluster, you can set up the namespace for the XTDB deployment:
+
+```bash
+kubectl create namespace xtdb-deployment
+```
+
+The GKE cluster is now ready for deployment,
+
+'''
+
+=== Deploying an example Kafka 
+
+To deploy a basic set of Kafka resources within GKE, you can make use of the `bitnami/kafka` Helm chart. Run the following command:
+
+```bash
+helm install kafka oci://registry-1.docker.io/bitnamicharts/kafka \
```

Comment:
If we wanted to make this guide more "production like" could use better kafka setup defaults.